### PR TITLE
[20108] Clarify Fast DDS Discovery Server Tutorial for Windows support

### PR DESCRIPTION
**Pull request by JesusPoderoso**
Fixes:
- https://github.com/ros2/ros2_documentation/issues/4037

### CODE REVIEW

File: source/Tutorials/Advanced/Discovery-Server/Discovery-Server.rst [hunk 1]
```diff
@@ -121,9 +131,22 @@ Launch talker node
...
+
+        .. code-block:: console
+
+            export ROS_DISCOVERY_SERVER=127.0.0.1:11811
+
+   .. group-tab:: Windows
+
+        .. code-block:: console
+
+            set export ROS_DISCOVERY_SERVER=127.0.0.1:11811
```
Comment (elianalf) [inline source/Tutorials/Advanced/Discovery-Server/Discovery-Server.rst]: I think this should be set ROS_DISCOVERY_SERVER=127.0.0.1:11811

**Review by elianalf — CHANGES_REQUESTED** [review]

File: source/Tutorials/Advanced/Discovery-Server/Discovery-Server.rst (hunk 1 above)
Comment (fujitatomoya) [reply source/Tutorials/Advanced/Discovery-Server/Discovery-Server.rst]:
```suggestion
            set ROS_DISCOVERY_SERVER=127.0.0.1:11811
```

**Review by fujitatomoya — CHANGES_REQUESTED** [review]
there are many places with `export`, those also needs to be updated accordingly?

**Comment by JesusPoderoso** [comment]
> there are many places with `export`, those also needs to be updated accordingly?

Sure, thanks @fujitatomoya!

**Review by elianalf — APPROVED** [review]

**Pull request merged**